tidb-distributed-execution-framework.md

```diff
@@ -39,13 +35,13 @@ In a database management system, in addition to the core transactional processin
 - Might need to be executed periodically, but at a low frequency.
 - If the resources are not properly controlled, they are prone to affect TP and AP tasks, lowering the database service quality.
 
-Enabling the TiDB backend task distributed execution framework can solve the above problems and has the following three advantages:
+Enabling the DXF can solve the above problems and has the following three advantages:
 
 - The framework provides unified capabilities for high scalability, high availability, and high performance.
 - The framework supports distributed execution of backend tasks, which can flexibly schedule the available computing resources of the entire TiDB cluster, thereby better utilizing the computing resources in a TiDB cluster.
 - The framework provides unified resource usage and management capabilities for both overall and individual backend tasks.
 
-Currently, for TiDB Self-Hosted, the TiDB backend task distributed execution framework supports the distributed execution of the `ADD INDEX` and `IMPORT INTO` statements. For TiDB Cloud, the `IMPORT INTO` statement is not applicable.
+Currently, for TiDB Self-Hosted, the DXF supports the distributed execution of the `ADD INDEX` and `IMPORT INTO` statements. For TiDB Cloud, the `IMPORT INTO` statement is not applicable until v7.2.
```

Comment:
The `IMPORT INTO` statement is not supported on TiDB Cloud.

Reply:
Updated